tests/cmsis_rtos_v1/threads/thread_apis/src/main.c

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <kernel.h>
+#include <cmsis_os.h>
+
+#define STACKSZ		512
+
+static int thread1_evidence;
```

Comment:
`thread1_evidence` can be named better? The variable is tracking the flow between two threads, so something on that note? Also, a comment here for what the variables are for, would be helpful

Reply:
renamed it to thread_yield_check.